ocs_ci/templates/openshift-infra/logging-deployment/clusterlogging_operator/cl-sub.yaml

```diff
@@ -4,7 +4,6 @@ metadata:
   name: cluster-logging
   namespace: openshift-logging
 spec:
-  channel: "4.2"
```

Comment:
I would keep the channel here and just replace it's contnet with VERISON_PLACEHOLDER like
```YAML
channel: "VERSION_PLACEHOLDER"
```
Then we easily know that it has to be replaced when used WDYT?